sdk/core/azure-core-test/src/main/java/module-info.java

```diff
@@ -16,7 +16,7 @@
 
     opens com.azure.core.test.implementation to com.fasterxml.jackson.databind, com.azure.core;
     opens com.azure.core.test.implementation.entities to com.fasterxml.jackson.databind, com.azure.core;
-    opens com.azure.core.test to com.fasterxml.jackson.databind;
+    opens com.azure.core.test to com.fasterxml.jackson.databind, org.junit.platform.commons;
```

Comment:
Shouldn't require an opens to JUnit in the `module-info`, should only be required in the POM where you made your other change.